pyclaragenomics/bin/evaluate_paf

```diff
@@ -62,6 +64,10 @@ def evaluate_paf(truth_paf_filepath, test_paf_filepath, pos_tolerance=400, skip_
     false_positive_count = 0
     false_negative_count = 0
 
+    print("Counted {} true overlaps".format(num_true_overlaps))
+
+    seen_test_overlap_keys = set()
```

Comment:
@tijyojwad this is added in case there are duplicate overlaps in the PAF file, which would mess up the statistics otherwise.